.gitignore

```diff
@@ -1,3 +1,6 @@
+# Ignore local Make configuration files:
+Makeconf
```

Comment:
Out of curiosity: Did you really use make in some way while implementing this PR?

Reply:
Yes, GNU Make is my favourite project management tool :)
But, agreed, not needed for JabRef. Removed the .gitignore lines.